src/unittest/TestParticle.py

```diff
@@ -30,7 +30,7 @@ def test0get(self):
         system.bc = bc.OrthorhombicBC(system.rng, (10.0, 10.0, 10.0))
         system.storage = espressopp.storage.DomainDecomposition(
             system=system,
-            nodeGrid=(1,1,1), cellGrid=(1,1,1))
+            nodeGrid=(1,1,1), cellGrid=(2,2,2))
```

Comment:
Why?

Reply:
The minimum amount of cells is two per axis, so that the Linked-Cell_List Algorithm with ghost cells works!